spacy/displacy/__init__.py

```diff
@@ -123,17 +123,18 @@ def app(environ, start_response):
 def parse_deps(orig_doc: Doc, options: Dict[str, Any] = {}) -> Dict[str, Any]:
     """Generate dependency parse in {'words': [], 'arcs': []} format.
 
-    doc (Doc): Document do parse.
+    orig_doc (Doc): Document to parse.
+    options (Dict[str, Any]): Dependency parse specific visualisation options.
     RETURNS (dict): Generated dependency parse keyed by words and arcs.
     """
-    doc = Doc(orig_doc.vocab).from_bytes(
+    orig_doc = Doc(orig_doc.vocab).from_bytes(
```

Comment:
I don't think you needed to rewrite all of these references to `doc`?

Reply:
Whoops. That was my IDE trying to be smart (and me being dumb) with Replace Symbol, I'll undo.